documentation/book/ref-jvm-options.adoc

```diff
@@ -105,3 +105,18 @@ The example configuration above will result in the following JVM options:
 ----
 
 NOTE: When neither of the two options (`-server` and `-XX`) is specified, the default Apache Kafka configuration of `KAFKA_JVM_PERFORMANCE_OPTS` will be used.
+
+== Garbage collector logging
+
+The `jvmOptions` section also allows you to enable and disable garbage collector (GC) logging.
```

Comment:
From a theoretical point of view, isn't better to use "or"?

Reply:
I copied from @laidan6000 suggestion so maybe it sounds better in English ... who knows? :-)